### PR TITLE
Add single W boson production for PbPb 5.36 TeV

### PR DESCRIPTION
**Pull request by stahlleiton**
This PR adds the Run3 PbPb Powheg inputs and genfragments for single W boson production decaying into lepton and neutrino.

@DickyChant

### CODE REVIEW

File: bin/Powheg/production/Run3/PbPb_5p36TeV/WToLNu/WmToENu_EPPS21.input [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+nubound  50000     ! number of bbarra calls to setup norm of upper bounding function
+icsimax  1         ! <= 100, number of csi subdivision when computing the upper bounds
+iymax    1         ! <= 100, number of y subdivision when computing the upper bounds
+xupbound 2d0       ! increase upper bound for radiation generation
+
+! OPTIONAL PARAMETERS
+
+renscfact  1d0     ! (default 1d0) ren scale factor: muren  = muref * renscfact
+facscfact  1d0     ! (default 1d0) fac scale factor: mufact = muref * facscfact
+testplots  1       ! (default 0, do not) do NLO and PWHG distributions
```
Comment (bbilin) [inline bin/Powheg/production/Run3/PbPb_5p36TeV/WToLNu/WmToENu_EPPS21.input:34]: Why ?

Comment (bbilin) [reply bin/Powheg/production/Run3/PbPb_5p36TeV/WToLNu/WmToENu_EPPS21.input:34]: (just curiosity.)

**Review by bbilin — APPROVED** [review]

File: bin/Powheg/production/Run3/PbPb_5p36TeV/WToLNu/WmToENu_EPPS21.input (hunk 1 above)
Comment (stahlleiton) [reply bin/Powheg/production/Run3/PbPb_5p36TeV/WToLNu/WmToENu_EPPS21.input:34]: I think I took it from the other Powheg cards. Do you recommend removing it?

Comment (DickyChant) [reply bin/Powheg/production/Run3/PbPb_5p36TeV/WToLNu/WmToENu_EPPS21.input:34]: I think only `testplots` is fishy, the other `muren` `mufact` scale factors are just normal

But for `testplots` IIRC it should be the topdraw hists that is computationally cheap and compact in size so perhaps fine

Comment (stahlleiton) [reply bin/Powheg/production/Run3/PbPb_5p36TeV/WToLNu/WmToENu_EPPS21.input:34]: ok, I will remove the testplots in the powheg cards

**Pull request merged**